pkg/controller/common/controller_context.go

```diff
@@ -55,7 +58,16 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 	kubeSharedInformer := informers.NewSharedInformerFactory(kubeClient, resyncPeriod()())
 	kubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), targetNamespace, nil)
 	openShiftConfigKubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), "openshift-config", nil)
-	apiExtSharedInformer := apiextinformers.NewSharedInformerFactory(apiExtClient, resyncPeriod()())
+	// filter out CRDs that do not have the MCO label
+	filterObjectChanges := func(opts *metav1.ListOptions) {
+		ls, err := labels.ConvertSelectorToLabelsMap(opts.LabelSelector)
+		if err != nil {
+			glog.Warningf("unable to convert selector %q to map: %v", opts.LabelSelector, err)
```

Comment:
Random question: do we want this as a warning? or a log? I'm not sure, just want to double check

Reply:
I think this should be a warning. If the cast fails, that means the ListOptions struct was incorrectly populated and needs to be fixed.

Reply:
could we rename this to assignFilterLabels() just for clarity then?